subprojects/platform-native/src/main/java/org/gradle/nativeplatform/tasks/InstallExecutable.java

```diff
@@ -133,6 +137,17 @@ public void setExecutable(File executable) {
         this.executable.set(executable);
     }
 
+    // Workaround for when the task is given an input file that doesn't exist
```

Comment:
This is confusing to me. I don't understand exactly what is being worked around. It looks like this method isn't even used. Also, how does this "cover" the "getSourceFile" method?

Reply:
Issue https://github.com/gradle/gradle/issues/2919 was opened to track this feature.